next-js/pages/_app.tsx

```diff
@@ -1,21 +1,54 @@
 import type { AppProps } from "next/app";
+import { useRouter } from "next/router";
 import { useEffect } from "react";
 import { GrowthBook, GrowthBookProvider } from "@growthbook/growthbook-react";
+import { Experiment, isURLTargeted, Result } from "@growthbook/growthbook";
 import Head from "next/head";
 
+const onExperimentViewed = (
+  experiment: Experiment<any>,
+  result: Result<any>
+) => {
+  const experimentId = experiment.key;
+  const variationId = result.key;
+
+  console.log("Viewed Experiment", {
+    experimentId,
+    variationId,
+    isURLTargeted: isURLTargeted(
```

Comment:
Don't need to include this. The tracking callback is only fired if the user is included in the experiment, so this will always be true.

Reply:
Ok - it was meant to illustrate the API to show users what is available. I can remove it

Reply:
Updated